lib/client/mfa.go

```diff
@@ -196,3 +196,45 @@ func MakeAuthenticateChallenge(protoChal *proto.MFAAuthenticateChallenge) *auth.
 
 	return chal
 }
+
+type TOTPChallenge struct {
+	QRCodeBytes []byte `json:"qrCodeBytes"`
+}
```

Comment:
```suggestion
type TOTPRegisterChallenge struct {
	QRCode []byte `json:"qrCode"`
}
```

so it matches the proto? (assuming you took the QRCode rename too)